src/registrar/admin.py

```diff
@@ -2594,28 +2849,71 @@ def queryset(self, request, queryset):
         ),
     )
 
+    # ------- Domain Information Fields
+
+    # --- Generic Org Type
+    @admin.display(description=_("Converted Generic Org Type"))
```

Comment:
**changes requested** @CocoByte we don't want this to show as "converted" anywhere for the users. This is not meant to be a user facing change and is meant to make sure users see the correct data and don't need to know how the sauce is made. Please change all the ones like this.